File: 666-web/src/main/webapp/jsp/AnglaJS/js/service/userService.js
//服务层
app.service('userService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../user/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../user/findPage?page='+page+'&rows='+rows);
	}
	//今日
	this.findToday=function(){
		return $http.get('../user/findToday');
	}
	//查询结果
	this.count=function () {
		return $http.get('../user/count');
	}

	//查询实体
	this.findOne=function(id){
		return $http.get('../user/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../user/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../user/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../user/delete?ids='+ids);
	}
	//搜索
	this.search=function( searchEntity){
		return $http.post('../user/search', searchEntity);
	}    	
});

File: 666-service/src/main/java/com/service/UserService.java
package com.service;
import pojo.User;

import java.util.List;





/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface UserService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<User> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	//public PageResult findPage(int pageNum, int pageSize);

	//今日
	public  List<User> findtoday();

	//计算结果
	public  List<User> coun();



	/**
	 * 增加
	*/
	public void add(User user);
	
	
	/**
	 * 修改
	 */
	public void update(User user);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public User findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] ids);

	//查询
	public List<User> find(User user);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	//public PageResult findPage(User user, int pageNum, int pageSize);
	
}
